test/integration/targets/forks/test_forks.yml

```diff
@@ -0,0 +1,229 @@
+---
+- hosts: localhosts
+  gather_facts: false
+  vars:
+    forksdir: ~/ansible_testing/forks.dir/
+  tasks:
+    - name: Clean forksdir '{{ forksdir }}'
+      file:
+        state: absent
+        path: '{{ forksdir }}'
+      ignore_errors: yes
+      run_once: yes
+    - name: Create forksdir '{{ forksdir }}'
+      file:
+        state: directory
+        path: '{{ forksdir }}'
+      run_once: yes
+    - block:
+      - name: "Test 1 (max forks: 3)"
+        command: python
```

Comment:
Instead of `python` this should be utilizing `ansible_python_interpreter`, potentially falling back to another python.

Reply:
ansible_playbook_python, since you are executing on "localhosts" (i assume connection = local)

Reply:
I have fixed this in the test case.